lib/modules/keywords.mjs

```diff
@@ -53,19 +52,22 @@ export async function validate2119Keywords (doc, { mode = MODES.NORMAL } = {}) {
             ref: 'https://www.rfc-editor.org/rfc/rfc7322.html#section-4.8.2'
           }))
         }
-      }
-
-      if (hasBoilerplate && !hasNonBoilerplateKeywords) {
-        result.push(new ValidationWarning('MISSING_REQLEVEL_KEYWORDS', 'An RFC2119 boilerplate is present but no keywords are used in the document.', {
+      } else if (!hasBoilerplate && doc.data.boilerplate.similar2119boilerplate) {
```

Comment:
This isn't a duplicate error - I don't agree with the change to `else if` with at least this case.
If someone has use MUST and has a non-conforming 2119 boilerplate that would match the "similar" regex, as the code is now, they will only be told "You don't have the boilerplate or a reference" and they'll say "I do too". If we tell them that they had a similar but incorrect string at the same time, they'll go "oh yeah" and fix the issue.